go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -67,9 +68,14 @@ func newNodeController(
 	nodeLister listers.NodeLister,
 	podLister listers.PodLister,
 ) (nodeController, error) {
+
+	if config.HybridOverlay.VXLANPort > 65535 {
```

Comment:
@JacobTanenbaum this part should already be verified in buildHybridOverlayConfig() in config.go right?

Reply:
It was but it was verified on the windows side again so I thought I would too. I removed it